### PR TITLE
[Feature] - CON-462 - Default Item language

### PR DESCRIPTION
**Pull request by 404passionFruit**
The default language on authoring mode for both passages or items should be customizable

https://oat-sa.atlassian.net/browse/CON-462
https://oat-sa.atlassian.net/browse/CON-280

- [ ]  https://github.com/oat-sa/generis/pull/860
- [ ]  https://github.com/oat-sa/tao-core/pull/2775
- [ ]  https://github.com/oat-sa/extension-tao-udir/pull/105

### CODE REVIEW

File: manifest.php [hunk 1]
```diff
@@ -43,12 +43,12 @@
     'name'        => 'taoQtiItem',
     'label'       => 'QTI item model',
     'license'     => 'GPL-2.0',
-    'version'     => '25.21.0',
+    'version'     => '25.21.1',
```
Comment (bziondik) [inline manifest.php:46]: should be 25.22.0 - it is feature

Comment (404passionFruit) [reply manifest.php:46]: You a right, good catch! Thank you

**Review by habemuscode — APPROVED** [review]
Take a loo at Hanna comment, the rest, fine for me

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

File: views/js/qtiCreator/model/Item.js [hunk 2]
```diff
@@ -18,14 +18,15 @@
...
+    'context',
     'taoQtiItem/qtiCreator/model/mixin/editable',
     'taoQtiItem/qtiCreator/model/mixin/editableContainer',
     'taoQtiItem/qtiItem/core/Item',
     'taoQtiItem/qtiCreator/model/Stylesheet',
     'taoQtiItem/qtiCreator/model/ResponseProcessing',
     'taoQtiItem/qtiCreator/model/variables/OutcomeDeclaration',
     'taoQtiItem/qtiCreator/model/feedbacks/ModalFeedback'
-], function(_, editable, editableContainer, Item, Stylesheet, ResponseProcessing, OutcomeDeclaration, ModalFeedback){
+], function(_, context, editable, editableContainer, Item, Stylesheet, ResponseProcessing, OutcomeDeclaration, ModalFeedback, config){
```
Comment (bziondik) [inline views/js/qtiCreator/model/Item.js:29]:
```suggestion
], function(_, context, editable, editableContainer, Item, Stylesheet, ResponseProcessing, OutcomeDeclaration, ModalFeedback){
```

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Pull request merged**